app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -271,9 +273,18 @@ class HomeFragment : Fragment() {
             if (requireContext().settings().showPocketRecommendationsFeature) {
                 val categories = components.core.pocketStoriesService.getStories()
                     .groupBy { story -> story.category }
-                    .map { (category, stories) -> PocketRecommendedStoriesCategory(category, stories) }
+                    .map { (category, stories) ->
+                        PocketRecommendedStoriesCategory(
+                            category,
+                            stories
+                        )
+                    }
```

Comment:
nit: For follow up PRs, lets try to format changes related to the PR. It make the diff difficult to read :)

Reply:
Ah my bad. I'd seen that I had formatted that file at some point and then forgot to revert the change. I'll leave this for now and be more careful in the future. Thanks for the call out

Reply:
No problem :)